samples/kotlin-mcp-client/src/main/kotlin/io/modelcontextprotocol/sample/client/MCPClient.kt

```diff
@@ -16,9 +16,18 @@ import kotlinx.io.buffered
 import kotlinx.serialization.json.JsonObject
 import kotlin.jvm.optionals.getOrNull
 
+import java.nio.file.Files
+import java.nio.file.Paths
+
+import java.io.File
+import io.modelcontextprotocol.kotlin.sdk.ReadResourceRequest
+import io.modelcontextprotocol.kotlin.sdk.TextResourceContents
+
 class MCPClient : AutoCloseable {
-    // Configures using the `ANTHROPIC_API_KEY` and `ANTHROPIC_AUTH_TOKEN` environment variables
-    private val anthropic = AnthropicOkHttpClient.fromEnv()
+
+    private val anthropic = AnthropicOkHttpClient.builder()
+        .apiKey(System.getenv("ANTHROPIC_API_KEY") )
+        .build()
```

Comment:
Why are these changes necessary? `AnthropicOkHttpClient.fromEnv()` looks for `ANTHROPIC_API_KEY` in the environment variables by default